pkg/eval/builtin_fn_styled_test.go

```diff
@@ -68,17 +68,26 @@ func TestStyledConcat(t *testing.T) {
 		That("print (styled-segment abc &bg-color=red)(styled-segment abc &fg-color=red)").Prints("\033[41mabc\033[m\033[31mabc\033[m"),
 		// segment+text
 		That("print (styled-segment abc &underlined=$true)(styled abc bright-cyan)").Prints("\033[4mabc\033[m\033[96mabc\033[m"),
-	)
-
-	Test(t,
+		// segment+num
+		That("print (float64 99)(styled-segment abc &blink)").Prints("99.0\033[5mabc\033[m"),
+		That("print (num 66)(styled-segment abc &blink)").Prints("66\033[5mabc\033[m"),
+		That("print (num 33)(styled-segment abc &blink)").Prints("33\033[5mabc\033[m"),
```

Comment:
Both cases test the `case int` branch. Did you mean to make one of them test a rational number?